Bottom: e38a3289eccd356e068edf795cde317e12ce91bb
Top:    847d456e3ddd47dfbefac8d1a190fb55b588f85e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 22:25:40 +0000

Refresh of add-hid_draw-parameter-to

---

diff --git a/src/hid/nelma/nelma.c b/src/hid/nelma/nelma.c
index 2295607..1f84a3a 100644
--- a/src/hid/nelma/nelma.c
+++ b/src/hid/nelma/nelma.c
@@ -729,7 +729,7 @@ nelma_do_export(HID_Attr_Val * options)
 /* *** PNG export (slightly modified code from PNG export HID) ************* */
 
 static int 
-nelma_set_layer(hidGC gc, const char *name, int group, int empty)
+nelma_set_layer (HID_DRAW *hid_draw, const char *name, int group, int empty)
 {
 	int             idx = (group >= 0 && group < max_group) ?
 	PCB->LayerGroups.Entries[group][0] : group;
